sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -719,13 +715,27 @@ MarginCalculationLauncher::synchronizeSuccesses(const std::vector<bool>& success
   context.broadcast(allSuccesses);
   return allSuccesses;
 }
+#endif
+
+std::string
+MarginCalculationLauncher::computeLoadIncreaseScenarioId(double variation) {
+  std::stringstream ss;
+  ss << LOAD_INCREASE << "-" << variation;
+  return ss.str();
+}
 
 void
 MarginCalculationLauncher::findOrLaunchLoadIncrease(const boost::shared_ptr<LoadIncrease>& loadIncrease,
                                                     const double variation,
+#ifdef _MPI_
```

Comment:
I would use static_cast<void>(minVariation) etc  and ifndef MPI to handle this case instead of having it in the method definition